### PR TITLE
AWS Widget

### PR DESCRIPTION
**Pull request by gutobortolozzo**
Adding type and subnet to AWS instance information.

### CODE REVIEW

File: src/ext/aws/components/Instances.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ var Instances = React.createClass({
             return (
                 <div key={instance.id} className={cssClass}>
                     {instance.name}
-                    {instance.state}
+					{instance.state} - {instance.type} - {instance.subnet}
```
Comment (plouc) [inline src/ext/aws/components/Instances.jsx]: tabs should be replaced by spaces

File: src/ext/aws/client.js [hunk 2]
```diff
@@ -42,30 +42,10 @@ module.exports = {
...
-
-                        instanceData.SecurityGroups.forEach(function (sg) {
-                            instance.securityGroups.push(sg.GroupId);
-                        });
-
-                        instance.tags = {};
-                        instanceData.Tags.forEach(function (tag) {
-                            instance.tags[tag.Key.toLowerCase()] = tag.Value;
+                            securityGroups:   [],
+							image: 			  instanceData.ImageId,
```
Comment (plouc) [inline src/ext/aws/client.js]: tabs should be replaced by spaces

**Comment by plouc** [comment]
@gutobortolozzo I know the parts involving security groups, tags, vpc look like dead code :/ but it's not, this call can be used to extract data from ec2 instances for other (upcoming) AWS widgets, can you please restore it ?

**Comment by gutobortolozzo** [comment]
@plouc I restored tags, vpc and security groups and also replaced tabs by spaces.

**Pull request merged**